mycore-base/src/main/java/org/mycore/frontend/cli/MCRCommandLineInterface.java

```diff
@@ -332,13 +330,12 @@ public static void getURI(String uri, String file) throws Exception {
     public static List<String> readCommandsRessource(String resource) throws IOException {
         final URL resourceURL = MCRClassTools.getClassLoader().getResource(resource);
         if (resourceURL == null) {
-            return Collections.emptyList();
+            throw new IOException("Resource URL is null!");
         }
-        try (InputStream is = resourceURL.openStream()) {
-            try (BufferedReader reader = new BufferedReader(new InputStreamReader(is, Charset.defaultCharset()))) {
+        try (BufferedReader reader = new BufferedReader(
+            new InputStreamReader(resourceURL.openStream(), Charset.defaultCharset()))) {
```

Comment:
may leak resourceURL.openStream() and do not use defaultCharset() for classpath resources but stick it to UTF8 from StandardCharsets class